model/derived/gasoutputs.go

```diff
@@ -12,6 +12,7 @@ import (
 
 type GasOutputs struct {
 	tableName          struct{} `pg:"derived_gas_outputs"` //nolint: structcheck,unused
+	Height             int64    `pg:",pk,use_zero,notnull"`
```

Comment:
will this change need a schema migration?

Reply:
added below, forgot in first commit.